README.md

```diff
@@ -1,295 +1,202 @@
-- `..` The directory in which your current directory is contained—referred to
-as the "**parent**" directory.
-
-You can supply any path to the `cd` command to nkellyegreenegate to that location.
-
-## Demonstrate Manipulating Files in the Command Line
-
-### Paths in Shell
-
-The path supplied to the `cd` command, for example `/Users/kellyegreene`, is
-known as an absolute path.
-
-Systems can use either *absolute* or *relative* paths.
-
-An absolute path is a path that points to the same location on the file system
-regardless of the working directory. They start with `/` ("forward slash") because
-that is the root of your file system.
-
-This is an absolute path: `/Users/kellyegreene`.
-
-A relative path is a path **relative** to the working directory of the user or
-application, so the full absolute path will not have to be given. They start with
-the name of a directory or a file.
-
-This is a relative path: `kellyegreene/Documents`.
-
-Paths use `/` to denote levels.
-
-How many levels are within the following path?
-
-`/Users/kellyegreene/Development/code/flatiron-school/mixtape-app`
-
-If you said 6 levels to the question above you are right! Knowing where you
-are in your terminal is very important.
-
-### Use `ls` to List Files in Shell
-
-In a new terminal, try this:
+Now try:
 
 ```bash
 $ ls
 ```
 
-You should see a list of all the files within your working directory.
-
-The command `ls` stands for "**l**i**s**t".
+You should see the file you just created, `hello_world.rb`, in the working
+directory. Note that this is an empty file and has nothing inside of it, because
+you just created it. 
 
-We can use flags on most unix commands to give more specific instructions. Most
-programs also accept flags, or options for execution.
-
-A flag is denotated by a `-` ("dash"). **Note:** *In some programs, options are
-passed directly to the command and not via flags.*
-
-A common flag that nearly all programs and commands accept is a standalone `h`,
-for "**h**elp".
+**Unix systems are _case sensitive_**! This is very important. Other systems
+such as Windows are case insensitive when it comes to referring to files. Unix
+systems is not like this. As such it is possible to have two or more files and
+directories with the same name but letters of different case.
 
 ```bash
-$ ls -h
+hello_world.rb
+HELLO_WORLD.rb
+hello_world.RB
 ```
 
-Single-character options can typically be combined with each other. For example,
-in the `ls` command, `h` is a suffix on the `l` flag meaning "**h**uman readable
-formats." They can be combined with `a` meaning "**a**ll information including.
-Try these three together:
-
-We can use the `ls -al` flag with ls so that we can see all of the files including
-hidden files in "long form".
-
-```bash
-$ ls -lah
-```
-And also:
+In the code snippet above, these are all distinct and separate files.
 
-```bash
-$ ls -l -a -h
-```
-Both are valid input options.
+### Making New Directories Using `mkdir`
```

Comment:
Before this should we do `cat`, take a look at that empty file and prove it's empty? Also, let's use `ls -l` to show the file size...it should be 0. We're reinforcing that flags do useful stuff.